lms/static/sass/course/_info.scss

```diff
@@ -224,7 +224,6 @@ div.info-wrapper {
           text-transform: none;
           background: url('#{$static-path}/images/calendar-icon.png') 0 center no-repeat;
           @include padding-left($baseline);
-          @include float(left);
```

Comment:
I'm nervous about this because it will affect the existing page. Do you remember why this was necessary? Can we target this change just to the _course-experience.scss file?